lib/galaxy/authnz/managers.py

```diff
@@ -152,7 +152,7 @@ def _parse_custos_config(self, config_xml):
             'client_id': config_xml.find('client_id').text,
             'client_secret': config_xml.find('client_secret').text,
             'redirect_uri': config_xml.find('redirect_uri').text,
-            'enable_idp_logout': asbool(config_xml.findtext('enable_idp_logout', 'false'))}
+            'enable_idp_logout': asbool(config_xml.findtext('enable_idp_logout', 'true'))}
```

Comment:
no it does not. tested locally with two instances. I believe this is because each instance has its own database with a different token so they are operating independently

Reply:
that is the preferred behaviour, right?